lib/sync/index.js

```diff
@@ -53,6 +67,7 @@ function stop(dataset_id, cb) {
   cb(null, {});
 }
 
+// Stop cloud data sync service for ALL datasets.
 function stopAll(cb) {
   syncUtil.doLog(syncUtil.SYNC_LOGGER, 'info', 'stopAll');
   
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove the useless trailing whitespaces at the end of this line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3ATrailingWhitespace)